.travis.yml

```diff
@@ -2,7 +2,10 @@ language: node_js
 os:
 - linux
 - osx
+- windows
 osx_image: xcode12
+env:
+  - YARN_GPG=no
```

Comment:
I'm not sure if this is necessarily safe? This comment seems to offer a solution: https://github.com/yarnpkg/yarn/issues/2012#issuecomment-532426947

Reply:
ah good find! I was just following [this old recommendation from the Travis forum](https://travis-ci.community/t/build-doesnt-finish-after-completing-tests/288/13). I'll make that change in the next day or two and see if the build still passes.

Reply:
@ryb73 oh now I remember now why this won't work. The problem isn't installing gpg - that's already done - the problem is that it leaves `gpg-agent` running and one of the many bugs in the Travis windows VMs (since 2018) is that leftover processes cause the build to hang after it has completed.

I added a `ps -ef` to my trial run just in case this happened:
https://travis-ci.com/github/reasonml-labs/decco/jobs/506294058#L219

As near as I can tell environment variables can't be different per platform, so we (and every other project using yarn on travis windows builds) are stuck without GPG.